server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java

```diff
@@ -23,14 +29,38 @@
     @Autowired
     ArticleRepository articleRepository;
 
+    @Autowired
+    private HttpServletRequest httpServletRequest;
```

Comment:
We are in a singleton it's not very nice to inject httpRequest here. You have to do that in each method which uses it